src/core/agent/services/keriaNotificationService.ts

```diff
@@ -746,7 +748,9 @@ class KeriaNotificationService extends AgentService {
                   });
               for (const notification of notifications) {
                 // @TODO: Delete other long running operations in linkedGroupRequests
-                await this.deleteNotificationRecordById(
+                await deleteNotificationRecordById(
```

Comment:
Actually, since we still have `deleteNotificationRecordById` in this file for the UI - we can still use it in here. So just directly call it.

However, can we update it to be:
```
  async deleteNotificationRecordById(
    id: string,
    route: NotificationRoute
  ): Promise<void> {
    return deleteNotificationRecordById(this.props.signifyClient, this.notificationStorage, id, route);
  }
```

Not sure if you have to rename one

Reply:
This way, implementation will only change in one place and we only need tests in one place.